scripts/skills/effects/rf_bone_plating_effect.nut

```diff
@@ -0,0 +1,34 @@
+this.rf_bone_plating_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		SoundOnProcced = [
```

Comment:
1. Is this a vanilla thing? I don't know for sure but it seems triggering these sounds is a new feature in Reforged. If that is correct then please make this a separate commit.
2. Instead of hard-coding these sounds in a custom array in `m` you can instead use the vanilla `::Const.Sound.ArmorBoneImpact` array directly in the call to `::Sound.play` below in this skill.

Reply:
Yes it's a new feature in reforged. Though inspired by legends